docs/resources.rst

```diff
@@ -71,22 +60,31 @@ Editor Integrations
 
 * Vim
 
-    * `Vim Solidity <https://github.com/tomlion/vim-solidity/>`_
+    * `Vim Solidity by Thesis <https://github.com/thesis/vim-solidity/>`_
+        Syntax highlighting for Solidity in Vim.
+
+    * `Vim Solidity by tomlion <https://github.com/tomlion/vim-solidity/>`_
         Plugin for the Vim editor providing syntax highlighting.
 
     * `Vim Syntastic <https://github.com/vim-syntastic/syntastic>`_
         Plugin for the Vim editor providing compile checking.
 
 * Visual Studio Code
 
-    * `Visual Studio Code extension <https://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
+    * `Ethereum Remix <https://marketplace.visualstudio.com/items?itemName=RemixProject.ethereum-remix>`_
+        Ethereum Remix plugins in VSCode
+
+    * `Solidity by Juan Blanco <https://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
         Solidity plugin for Microsoft Visual Studio Code that includes syntax highlighting and the Solidity compiler.
 
+    * `Solidity by Nomic Foundation <https://marketplace.visualstudio.com/items?itemName=NomicFoundation.hardhat-solidity>`_
```

Comment:
It's technically called "Solidity" in the marketplace (probably for better positioning in search results?) but seems clearer to call it Hardhat. Especially when we have Truffle on this list too. [They call it that themselves](https://hardhat.org/hardhat-vscode).
```suggestion
    * `Hardhat for VS Code by Nomic Foundation <https://marketplace.visualstudio.com/items?itemName=NomicFoundation.hardhat-solidity>`_
```

Reply:
Tend to agree with cameel here, might be easier for people to differentiate between Remix, Truffle and Hardhat plugins this way. Unless we strictly want to use the official names from the VS Marketplace.